breakpoints/getCallbackFromUserFriendlyCallbackArgument.js

```diff
@@ -1,25 +1,28 @@
 import {runWithBreakpointsDisabled} from "./debugObj"
 
-function debuggerFunction(details){
-    debugger;
+function getDebuggerFunction() {
+    var debuggerFunc = function() {
+        debugger;
+    }
+    debuggerFunc.callbackType = "debugger";
```

Comment:
What is the point of introduction the `getDebuggerFunction` function? Wouldn't renaming the property to `callbackType` be enough?